src/components/tools/link-indicator.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { observer } from "mobx-react";
+import { GeometryMetadataModelType } from "../../models/tools/geometry/geometry-content";
+import { TableMetadataModelType } from "../../models/tools/table/table-content";
+import { findMetadata } from "../../models/tools/tool-types";
+import { IconButton } from "../utilities/icon-button";
+
+import "./link-indicator.sass";
+
+interface IProps {
+  type: string;
+  id: string;
+}
+
+@observer
+export class LinkIndicatorComponent extends React.Component<IProps>{
+
+  public render() {
+    const { id, type } = this.props;
+    const metadata = findMetadata(type, id);
+    const isGeometry = this.props.type === "Geometry";
+    const isTable = this.props.type === "Table";
+    const geometryLinkCount = metadata as GeometryMetadataModelType;
+    const tableLinkCount = metadata as TableMetadataModelType;
+    const linkCount = Math.max(isGeometry ? geometryLinkCount.linkedTableCount
```

Comment:
```suggestion
    const linkCount = Math.max(3, isGeometry ? geometryLinkCount.linkedTableCount
```
Math.max() doesn't do anything useful if you only pass it a single argument. The point here is to get the correct icon when there are more than three links.

Reply:
I believe `Math.max()` returns the largest of the integers provided? Trying this out also it set all of the icons to the third icon. I've replaced this with the following:
```
const linkCount = isGeometry ? (geometryMetadata.linkedTableCount <= 3 ? geometryMetadata.linkedTableCount : 3)
                                 : isTable ? (tableMetadata.linkCount <= 3 ? tableMetadata.linkCount : 3)
                                           : 0;
```

Reply:
🤦‍♂ Which is why it should be `Math.min()` not `Math.max()`.